src/calculateRentalCost.js

```diff
@@ -28,7 +28,17 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  if (days >= 1 && days < 3) {
```

Comment:
let's introduce more variables 
create a variable for days * 40 (you use it 3 times)
and variables for 20 and 50